app/site/content/_index.md

```diff
@@ -1,5 +1,8 @@
 ---
-title: "Linear Algebra"
+title: "Course Home"
+course_title: "Linear Algebra"
+course_image_url: "images/course_image.jpg"
+course_description: "This is a basic subject on matrix theory and linear algebra. Emphasis is given to topics that will be useful in other disciplines, including systems of equations, vector spaces, determinants, eigenvalues, similarity, and positive definite matrices."
```

Comment:
It's rendered in the template here:

```
<p class="text-white-mobile">{{ .Params.course_description }}</p>
```
If you put HTML into the course_description front matter variable, it will be rendered as text currently.  If we expect HTML to be put in here, we can pipe it to "safeHTML" like so:

```
<p class="text-white-mobile">{{ .Params.course_description | safeHTML }}</p>
```

I wasn't sure if we'd want to allow HTML in this field or not so I didn't include it, but we can decide to do so in the future if we want.